pkg/logs/input/kubernetes/launcher.go

```diff
@@ -88,7 +90,15 @@ func (l *Launcher) run() {
 	for {
 		select {
 		case service := <-l.addedServices:
-			l.addSource(service)
+			exp := &backoff.ExponentialBackOff{
+				InitialInterval:     500 * time.Millisecond,
```

Comment:
Even Kubelet has a transient failure, it should recover quite fast. I'd say Max: 30s / MaxInterval: 5s.